lesson-one/examples/hello-world-Nathan.html

```diff
@@ -0,0 +1,17 @@
+<html>
+    <head>
+        <title>Hello world page Nathan</title>
+    </head>
+    <style>
+        h1 {color :red;}
+        p {color :white;}
+	p.heart {color :pink; font-size: 100%; opacity :0.5;}
+	body {background-image: url("http://images.photowall.com/products/44352/rural-forest.jpg?h=650&q=90&")}
+    </style>
+    <body>
+        <h1> &#10084; Hello world!!</h1>
+		<p class="heart"> &#10084; &#10084; &#10084; &#10084; &#10084; &#10084; </font>
```

Comment:
Check this line here. You've got an opening `<p>` and then you try to close it with a `</font>`